receiver/hostmetricsreceiver/internal/scraper/virtualmemoryscraper/virtualmemory_constants.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package virtualmemoryscraper
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+// virtual memory metric constants
+
+const (
+	typeLabelName      = "type"
+	directionLabelName = "direction"
+)
+
+const (
+	majorTypeLabelValue    = "major"
+	inDirectionLabelValue  = "page_in"
+	outDirectionLabelValue = "page_out"
+)
+
+var metricPageFileUsedDescriptor = createMetricPageFileUsedDescriptor()
+
+func createMetricPageFileUsedDescriptor() pdata.MetricDescriptor {
+	descriptor := pdata.NewMetricDescriptor()
+	descriptor.InitEmpty()
+	descriptor.SetName("host/pagefile/percent_used")
```

Comment:
What do you think about this? Would make it in line with potential cpu/memory utilization metrics.

```suggestion
	descriptor.SetName("host/pagefile/utilization")
```

Reply:
👍 I wasn't sure if "utilization" would make sense for all metrics where we calculate percentages, but I can't think of any cases where it doesn't so I'll switch it over

Reply:
Note I changed approach here so am generating a "usage" instead of "utilization" stat now. I'll come back and add derived "utilization" stats for all relevant metrics in a later PR.